.github/workflows/prepare-for-prod-rollback.yml

```
@@ -89,13 +89,34 @@
+          cd rudderstack-operator
+          git checkout -b dedicated-transformer-rollback-${{ steps.target-version.outputs.tag_name }}
+
+          cd operator-helm/valuefiles
+          yq eval -i ".transformer.image.version=\"${{ steps.target-version.outputs.tag_name }}\"" values.yaml
+          git add values.yaml
+
+          git commit -m "chore: rollback dedicated transformer default version to ${{ steps.target-version.outputs.tag_name }}"
           git push -u origin dedicated-transformer-rollback-${{ steps.target-version.outputs.tag_name }}
 
-          gh pr create --fill
+          gh pr create --fill
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)